docs/deprecations.md

```diff
@@ -149,3 +149,47 @@ Please use `onUnauthenticatedRequest` instead. `allowUnauthenticated` will be re
 ## `bin/serverless`
 
 Please use `bin/serverless.js` instead. `bin/serverless` will be removed with v2.0.0
+
+## To Disable specific Deprecations
+
+To disable specific deprecations set `SLS_DEPRECATION_DISABLE` environment variable.
+
+Setting `SLS_DEPRECATION_DISABLE=*` will disable all deprecations.
+If you want to disable specific depreications set `SLS_DEPRECATION_DISABLE=CODE1,CODE2`.
+
+## List of Deprecation Codes
```

Comment:
Instead of listing the codes as a separate section, I believe it might be better to add Deprecation code: `CODE` in each existing section, for example:

```
<a name="AWS_ALB_ALLOW_UNAUTHENTICATED"><div>&nbsp;</div></a>

## AWS ALB `allowUnauthenticated`

Deprecation code: `AWS_ALB_ALLOW_UNAUTHENTICATED`

Please use `onUnauthenticatedRequest` instead. `allowUnauthenticated` will be removed with v2.0.0
```

Reply:
Thanks for your feedback, I have made the changes according to them.